eng/build.sh

```diff
@@ -402,6 +405,16 @@ while [[ $# > 0 ]]; do
       shift 1
       ;;
 
+     -packagerid)
+      if [ -z ${2+x} ]; then
+        echo "No value for packagerid is supplied. See help (--help) for supported values." 1>&2
+        exit 1
+      fi
+      packageRid="$(echo "$2" | tr "[:upper:]" "[:lower:]")"
+      arguments="$arguments /p:PackageRid=$packageRid"
```

Comment:
If we use `export __DistroRid="$packageRid"` here and call `initDistroRid` conditionally: `if [ -z "$packageRid" ]`. This way changes in `init-distro-rid.sh` won't be needed.